`chunkExtractor.getScriptElements` returns nested array in 5.13.1

## 🐛 Bug Report

Public API for `chunkExtractor.getScriptElements` is described as being an Array of React <script> Elements. As of version `5.13.1` it returns an array who's first element is also an array.

## To Reproduce

Unfortunately I cannot share my private repo. But the recent change to this function was the cause of this error. https://github.com/gregberge/loadable-components/blob/8cf392d124100763921226bdfcac10d350b64b34/packages/server/src/ChunkExtractor.js#L300 

Previously it returned a single element, but is now returning an array that isn't getting spread in this function 

https://github.com/gregberge/loadable-components/blob/8cf392d124100763921226bdfcac10d350b64b34/packages/server/src/ChunkExtractor.js#L384

## Expected behavior

`chunkExtractor.getScriptElements` returns an array of react script elements.

```bash

```

@theKashey - tried forking and install but I keep seeing this error: 
```
FAIL  packages/server/src/ChunkExtractor.test.js
  ● Test suite failed to run

    Cannot find module '@loadable/component' from 'sharedInternals.js'

    However, Jest was able to find:
    	'./sharedInternals.js'
```

Any ideas?

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -40,22 +40,28 @@ _Before_ submitting a pull request, please make sure the following is done…
     yarn --version
     ```
 
-4.  If you've added code that should be tested, add tests. You can use watch mode that continuously transforms changed files to make your life easier.
+4. Run `yarn build` to bootstrap packages.
+
+    ```sh
+    yarn build
+    ```
+
+5.  If you've added code that should be tested, add tests. You can use watch mode that continuously transforms changed files to make your life easier.
 
     ```sh
     # in the background
     yarn run dev
     ```
 
-5.  If you've changed APIs, update the documentation.
+6.  If you've changed APIs, update the documentation.
 
-6.  Ensure the linting is good via `yarn lint`.
+7.  Ensure the linting is good via `yarn lint`.
 
     ```sh-session
     $ yarn lint
     ```
 
-7.  Ensure the test suite passes via `yarn test`.
+8.  Ensure the test suite passes via `yarn test`.
 
     ```sh-session
     $ yarn test
@@ -63,7 +69,7 @@ _Before_ submitting a pull request, please make sure the following is done…
 
 #### Testing with your own project
 
-You can use `yarn run release-from-git` to create releases as tags on github. This requires that:
+You can use `yarn run release-to-git` to create releases as tags on github. This requires that:
 
 - Your git remote (where you want to publish the tags) is `origin`
 - Your commit messages follow the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) specification. For example: `feat: add timeout option`.

--- a/packages/server/src/ChunkExtractor.js
+++ b/packages/server/src/ChunkExtractor.js
@@ -297,7 +297,7 @@ class ChunkExtractor {
     ].join('')
   }
 
-  getRequiredChunksScriptElement(extraProps) {
+  getRequiredChunksScriptElements(extraProps) {
     const id = getRequiredChunkKey(this.namespace)
     const props = {
       type: 'application/json',
@@ -381,14 +381,14 @@ class ChunkExtractor {
   }
 
   getScriptElements(extraProps = {}) {
-    const requiredScriptElement = this.getRequiredChunksScriptElement(
+    const requiredScriptElements = this.getRequiredChunksScriptElements(
       extraProps,
     )
     const mainAssets = this.getMainAssets('script')
     const assetsScriptElements = mainAssets.map(asset =>
       assetToScriptElement(asset, extraProps),
     )
-    return [requiredScriptElement, ...assetsScriptElements]
+    return [...requiredScriptElements, ...assetsScriptElements]
   }
 
   getCssString() {

--- a/packages/server/src/ChunkExtractor.test.js
+++ b/packages/server/src/ChunkExtractor.test.js
@@ -133,26 +133,24 @@ describe('ChunkExtrator', () => {
       })
       expect(extractor.getScriptElements()).toMatchInlineSnapshot(`
         Array [
-          Array [
-            <script
-              dangerouslySetInnerHTML={
-                Object {
-                  "__html": "[]",
-                }
+          <script
+            dangerouslySetInnerHTML={
+              Object {
+                "__html": "[]",
               }
-              id="testapp__LOADABLE_REQUIRED_CHUNKS__"
-              type="application/json"
-            />,
-            <script
-              dangerouslySetInnerHTML={
-                Object {
-                  "__html": "{\\"namedChunks\\":[]}",
-                }
+            }
+            id="testapp__LOADABLE_REQUIRED_CHUNKS__"
+            type="application/json"
+          />,
+          <script
+            dangerouslySetInnerHTML={
+              Object {
+                "__html": "{\\"namedChunks\\":[]}",
               }
-              id="testapp__LOADABLE_REQUIRED_CHUNKS___ext"
-              type="application/json"
-            />,
-          ],
+            }
+            id="testapp__LOADABLE_REQUIRED_CHUNKS___ext"
+            type="application/json"
+          />,
           <script
             async={true}
             data-chunk="main"
@@ -165,26 +163,24 @@ describe('ChunkExtrator', () => {
     it('should return main script tag without chunk', () => {
       expect(extractor.getScriptElements()).toMatchInlineSnapshot(`
         Array [
-          Array [
-            <script
-              dangerouslySetInnerHTML={
-                Object {
-                  "__html": "[]",
-                }
+          <script
+            dangerouslySetInnerHTML={
+              Object {
+                "__html": "[]",
               }
-              id="__LOADABLE_REQUIRED_CHUNKS__"
-              type="application/json"
-            />,
-            <script
-              dangerouslySetInnerHTML={
-                Object {
-                  "__html": "{\\"namedChunks\\":[]}",
-                }
+            }
+            id="__LOADABLE_REQUIRED_CHUNKS__"
+            type="application/json"
+          />,
+          <script
+            dangerouslySetInnerHTML={
+              Object {
+                "__html": "{\\"namedChunks\\":[]}",
               }
-              id="__LOADABLE_REQUIRED_CHUNKS___ext"
-              type="application/json"
-            />,
-          ],
+            }
+            id="__LOADABLE_REQUIRED_CHUNKS___ext"
+            type="application/json"
+          />,
           <script
             async={true}
             data-chunk="main"
@@ -198,26 +194,24 @@ describe('ChunkExtrator', () => {
       extractor.addChunk('letters-A')
       expect(extractor.getScriptElements()).toMatchInlineSnapshot(`
         Array [
-          Array [
-            <script
-              dangerouslySetInnerHTML={
-                Object {
-                  "__html": "[\\"chunk-0-for-letters-A\\"]",
-                }
+          <script
+            dangerouslySetInnerHTML={
+              Object {
+                "__html": "[\\"chunk-0-for-letters-A\\"]",
               }
-              id="__LOADABLE_REQUIRED_CHUNKS__"
-              type="application/json"
-            />,
-            <script
-              dangerouslySetInnerHTML={
-                Object {
-                  "__html": "{\\"namedChunks\\":[\\"letters-A\\"]}",
-                }
+            }
+            id="__LOADABLE_REQUIRED_CHUNKS__"
+            type="application/json"
+          />,
+          <script
+            dangerouslySetInnerHTML={
+              Object {
+                "__html": "{\\"namedChunks\\":[\\"letters-A\\"]}",
               }
-              id="__LOADABLE_REQUIRED_CHUNKS___ext"
-              type="application/json"
-            />,
-          ],
+            }
+            id="__LOADABLE_REQUIRED_CHUNKS___ext"
+            type="application/json"
+          />,
           <script
             async={true}
             data-chunk="main"
@@ -236,26 +230,24 @@ describe('ChunkExtrator', () => {
       extractor.addChunk('letters-E')
       expect(extractor.getScriptElements()).toMatchInlineSnapshot(`
         Array [
-          Array [
-            <script
-              dangerouslySetInnerHTML={
-                Object {
-                  "__html": "[\\"letters-E\\"]",
-                }
+          <script
+            dangerouslySetInnerHTML={
+              Object {
+                "__html": "[\\"letters-E\\"]",
               }
-              id="__LOADABLE_REQUIRED_CHUNKS__"
-              type="application/json"
-            />,
-            <script
-              dangerouslySetInnerHTML={
-                Object {
-                  "__html": "{\\"namedChunks\\":[\\"letters-E\\"]}",
-                }
+            }
+            id="__LOADABLE_REQUIRED_CHUNKS__"
+            type="application/json"
+          />,
+          <script
+            dangerouslySetInnerHTML={
+              Object {
+                "__html": "{\\"namedChunks\\":[\\"letters-E\\"]}",
               }
-              id="__LOADABLE_REQUIRED_CHUNKS___ext"
-              type="application/json"
-            />,
-          ],
+            }
+            id="__LOADABLE_REQUIRED_CHUNKS___ext"
+            type="application/json"
+          />,
           <script
             async={true}
             data-chunk="main"
@@ -275,28 +267,26 @@ describe('ChunkExtrator', () => {
       expect(extractor.getScriptElements({ nonce: 'testnonce' }))
         .toMatchInlineSnapshot(`
         Array [
-          Array [
-            <script
-              dangerouslySetInnerHTML={
-                Object {
-                  "__html": "[\\"chunk-0-for-letters-A\\"]",
-                }
+          <script
+            dangerouslySetInnerHTML={
+              Object {
+                "__html": "[\\"chunk-0-for-letters-A\\"]",
               }
-              id="__LOADABLE_REQUIRED_CHUNKS__"
-              nonce="testnonce"
-              type="application/json"
-            />,
-            <script
-              dangerouslySetInnerHTML={
-                Object {
-                  "__html": "{\\"namedChunks\\":[\\"letters-A\\"]}",
-                }
+            }
+            id="__LOADABLE_REQUIRED_CHUNKS__"
+            nonce="testnonce"
+            type="application/json"
+          />,
+          <script
+            dangerouslySetInnerHTML={
+              Object {
+                "__html": "{\\"namedChunks\\":[\\"letters-A\\"]}",
               }
-              id="__LOADABLE_REQUIRED_CHUNKS___ext"
-              nonce="testnonce"
-              type="application/json"
-            />,
-          ],
+            }
+            id="__LOADABLE_REQUIRED_CHUNKS___ext"
+            nonce="testnonce"
+            type="application/json"
+          />,
           <script
             async={true}
             data-chunk="main"
@@ -321,28 +311,26 @@ describe('ChunkExtrator', () => {
         }),
       ).toMatchInlineSnapshot(`
         Array [
-          Array [
-            <script
-              dangerouslySetInnerHTML={
-                Object {
-                  "__html": "[\\"chunk-0-for-letters-A\\"]",
-                }
+          <script
+            dangerouslySetInnerHTML={
+              Object {
+                "__html": "[\\"chunk-0-for-letters-A\\"]",
               }
-              id="__LOADABLE_REQUIRED_CHUNKS__"
-              nonce="anonymous"
-              type="application/json"
-            />,
-            <script
-              dangerouslySetInnerHTML={
-                Object {
-                  "__html": "{\\"namedChunks\\":[\\"letters-A\\"]}",
-                }
+            }
+            id="__LOADABLE_REQUIRED_CHUNKS__"
+            nonce="anonymous"
+            type="application/json"
+          />,
+          <script
+            dangerouslySetInnerHTML={
+              Object {
+                "__html": "{\\"namedChunks\\":[\\"letters-A\\"]}",
               }
-              id="__LOADABLE_REQUIRED_CHUNKS___ext"
-              nonce="anonymous"
-              type="application/json"
-            />,
-          ],
+            }
+            id="__LOADABLE_REQUIRED_CHUNKS___ext"
+            nonce="anonymous"
+            type="application/json"
+          />,
           <script
             async={true}
             data-chunk="main"
@@ -368,26 +356,24 @@ describe('ChunkExtrator', () => {
 
       expect(extractor.getScriptElements()).toMatchInlineSnapshot(`
         Array [
-          Array [
-            <script
-              dangerouslySetInnerHTML={
-                Object {
-                  "__html": "[]",
-                }
+          <script
+            dangerouslySetInnerHTML={
+              Object {
+                "__html": "[]",
               }
-              id="__LOADABLE_REQUIRED_CHUNKS__"
-              type="application/json"
-            />,
-            <script
-              dangerouslySetInnerHTML={
-                Object {
-                  "__html": "{\\"namedChunks\\":[]}",
-                }
+            }
+            id="__LOADABLE_REQUIRED_CHUNKS__"
+            type="application/json"
+          />,
+          <script
+            dangerouslySetInnerHTML={
+              Object {
+                "__html": "{\\"namedChunks\\":[]}",
               }
-              id="__LOADABLE_REQUIRED_CHUNKS___ext"
-              type="application/json"
-            />,
-          ],
+            }
+            id="__LOADABLE_REQUIRED_CHUNKS___ext"
+            type="application/json"
+          />,
           <script
             async={true}
             data-chunk="main"

